### PR TITLE
becksteinlab Gromacs environment

### LINKED ISSUE #79: install Gromacs in Travis CI
Install
- [ ] Gromacs 5.1.x
- [ ] Gromacs 4.6.7
so that we can run a build matrix.

Needed to complete #39.

See Also: PR Becksteinlab/MDPOW#44 and issue Becksteinlab/MDPOW#41.

### PR DESCRIPTION
**Pull request by ianmkenney**
Adding the becksteinlab Gromacs conda packages for testing on TravisCI. Multiple versions of Gromacs can be used and submitted as matrix jobs.

Fixes #79.

### CODE REVIEW

**Comment by ianmkenney** [comment]
@orbeckst it looks like Gromacs installs correctly for both 5.1.2 and 4.6.7. Take a look at the changes and let me know what you think

File: .travis.yml [hunk 1]
```diff
@@ -28,15 +32,31 @@ install:
...
   # install test tools
   - pip install codecov
   - pip install pytest-cov
   - pip install pytest-pep8
   - pip install -e .
 
+  # source gromacs
+  - source $($HOME/get_gmx $GMX_VERSION)
+
+  # test gromacs output before merging pull request
```
Comment (orbeckst) [inline .travis.yml]: obviously, remove before merge

**Comment by orbeckst** [comment]
Very good, just clean up, and I will merge.

**Comment by orbeckst** [comment]
@ianmkenney , I am handling the merge, you're done on this PR. Good job!

**Pull request merged**